src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -871,6 +871,18 @@ public AsyncIndex getAsyncIndexConnection(String indexName) throws PineconeValid
         return new AsyncIndex(connection, indexName);
     }
 
+    /**
+     * A method to create and return a new instance of the {@link Inference} client.
+     * <p>
+     * This method initializes the Inference client using the current ApiClient
+     * from the {@link ManageIndexesApi}. The Inference client can then be used
+     * to interact with Pinecone's embedding API.
```

Comment:
Nit: it's the Inference API, not the embedding API.

Reply:
Yes, needs to be updated!